docs/pages/integrations/django/index.md

```diff
@@ -129,6 +129,108 @@ Class-based views are also supported. Specify dependencies in the class `__init_
 
 For more examples, see the [Wireup Django integration tests](https://github.com/maldoinc/wireup/tree/master/test/integration/django/view.py).
 
+### Non core-django views (eg, Django REST framework)
```

Comment:
Here we can mention Django Ninja as well and also state that this should work for any django-based framework as long as they have the same AppConfig and middleware mechanism as Django.